src/imp/passwd.h

```diff
@@ -0,0 +1,28 @@
+/************************************************************\
+ * Copyright 2020 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef HAVE_IMP_PASSWD_H
+#define HAVE_IMP_PASSWD_H 1
+
+#include <pwd.h>
+#include <sys/types.h>
+
+/*
+ *  Return a copy of the passwd entry for UID
+ *  Caller must free with passwd_destroy()
+ */
```

Comment:
Perhaps mention that some fields are not copied.

Reply:
Yeah, sorry meant to update to copying all fields before pushing this up. Got too excited that it was working.. next push will have a fix.